styles.css

```diff
@@ -0,0 +1,768 @@
+  justify-content: flex-end;
+  width: 100%;
+  align-items: center;
+  margin-right: 5rem;
+}
+
+.nav2-link {
+  padding: 0 1em;
+  color: #668b9a;
+}
+
+.languages {
+  background-color: #212529;
+  color: #668b9a;
+  margin: 0 1rem;
+}
+
+.fa-facebook-f,
+.fa-twitter {
+  color: #fff;
+}
+
+.nav1-item {
+  color: #668b9a;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: end;
+}
+
+.menu-wrap {
+  background-color: #f8f9fa;
+}
+
+.about {
+  color: red;
+}
+
+.campaign {
+  color: red;
+  border: 2px solid red;
+  padding: 0.8rem 0.2rem;
+}
+
+.showcase {
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+  margin: auto;
+  padding: 0 3rem;
+}
+
+.showcase::before,
+.about-header::before {
+  content: "";
+  background-image: url('img/system3.jpg');
+  background-size: cover;
+  position: absolute;
+  top: -18px;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  opacity: 0.4;
+
+  /* padding-bottom: 10rem; */
+}
+
+.showcase h1 {
+  position: relative;
+  left: 0;
+  color: #e85242;
+  font-size: 3rem;
+  line-height: 1.2;
+  padding-top: 10rem;
+  margin-left: 0;
+  max-width: 45rem;
+  font-weight: 600;
+}
+
+.about-header {
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  padding: 3rem;
+  max-width: 100%;
+  margin: auto;
+}
+
+.about-header h1 {
+  position: relative;
+  text-align: center;
+  align-self: center;
+  color: #e85242;
+  font-size: 3rem;
+  line-height: 1.2;
+
+  /* padding-top: 5rem; */
+  margin-top: 5rem;
+  max-width: 45rem;
+  font-weight: 600;
+}
+
+.about-para1 {
+  position: relative;
+  border: 2px solid white;
+  max-width: 45rem;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  background-color: #fff;
+  align-self: center;
+}
+
+.about-para2 {
+  position: relative;
+  padding-top: 5rem;
+  align-self: center;
+  color: #e85242;
+  font-weight: 900;
+  text-align: center;
+}
+
+.about-email {
+  position: relative;
+  padding-bottom: 10rem;
+  font-size: medium;
+  font-weight: 900;
+  align-self: center;
+  color: #e85242;
+  text-align: center;
+}
+
+.about-hr {
+  background-color: rgb(199, 194, 194);
+  width: 100%;
+}
+
+.about-logo img {
+  width: 10rem;
+}
+
+.about-logo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 2rem;
+}
+
+.logo-text {
+  font-weight: 600;
+  margin: 0 2rem;
+  font-size: small;
+}
+
+.about-logo p {
+  padding: 3rem 0;
+  text-align: center;
+}
+
+.about-logo h2 {
+  padding-top: 1rem;
+  font-weight: bolder;
+}
+
+.about-courses {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-courses h2 {
+  font-weight: bolder;
+}
+
+.about-img-courses {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+
+  /* border: 1px solid red; */
+  height: 20rem;
+  align-items: center;
+  top: 0;
+}
```

Comment:
- [ ] We have some small adjustments to do here at the courses section. Let's look at the screenshots below before we highlight what we need to address:

| The Provided Design |
| --- |
| ![main-pg-design](https://user-images.githubusercontent.com/45002654/133525566-f0da122e-aa09-4ab5-be9b-dfce335a6cfc.PNG) |

| The Implementation |
| --- |
| ![main-pg-imp](https://user-images.githubusercontent.com/45002654/133525531-a6b7afeb-8935-457f-92ef-d985d1cfffda.PNG) |

## Let's address the following
- [ ] Margins between the icons and the text below them. Add some, please.
- [ ] Text should be centered.
- [ ] Adjust the height and padding the containers. They have very big empty spaces at the bottom destroying the aesthetic of this section.